docs/layers/hexagon-layer.md

```diff
@@ -75,6 +77,15 @@ to number of counts by passing in an arbitrary color domain. This property is ex
 Hexagon color ranges as an array of colors formatted as `[[255, 255, 255, 255]]`. Default is
 [colorbrewer](http://colorbrewer2.org/#type=sequential&scheme=YlOrRd&n=6) `6-class YlOrRd`.
 
+##### `colorValueAccessor` (Function, optional)
```

Comment:
1) This function is optional and default is `undefined`? Can this be clarified a bit more?

2) About the naming: we never call our accessors `...Accessor`. If this is a normal accessor, can we just call this `getColorValue`?

On the other hand, if it is not a normal accessor and we want to make that clear, then let's at least not call it `...Accessor`.

Assuming we are trying to match the 'hexagonAggregator` naming pattern, maybe this could be `colorValueExtractor` or `colorValueGetter` or similar.